etc/profile-a-l/jami.profile

```diff
@@ -0,0 +1,17 @@
+# Firejail profile for jami
+# This file is overwritten after every install/update
```

Comment:
Not a must, but if you can add a fitting, short 'Decription: ...' of the program, that always helps users to get an idea of what this profile is about.

Reply:
Is "Proprietary Hoppscotch" going too far?